security/cmd/node_agent/server.go

```diff
@@ -0,0 +1,221 @@
+// Copyright 2018 Istio Authors
```

Comment:
I would put everything other than main.go to a package, i.e., node_agent/server/server.go. And for long term, all stuff should go to security/pkg/..., and security/cmd/ should only have main.go

Reply:
Absolutely, the cmd/ path should contains nothing except main.go. I was thinking to move things to security/pkg later since this current PR still is a prototype, not fully merged with the other node_agent/ implementation yet.

Do you want to move lib code to in this PR? That's also fine to me.